src/GitHash.hs

```diff
@@ -144,12 +144,12 @@ getGitFilesRegular git = do
       Right hdRef -> do
```

Comment:
Please add a changelog entry and version bump for this change.

Reply:
I thought this would be a minor bump. Am I correct?

Reply:
Yes, minor bump is perfect.